### PR TITLE
Implemented typography

### PR DESCRIPTION
**Pull request by gonuguntla**

### CODE REVIEW

File: app/javascript/components/Home/Home.jsx [hunk 1]
```diff
@@ -92,6 +92,7 @@ export default function Home(props) {
                 id="mln-campaign-hero"
                 text="Welcome to MyLibraryNYC"
                 color="ui.white"
+                className="noTextTransform"
```
Comment (gonuguntla) [inline app/javascript/components/Home/Home.jsx]: I added noTextTransform class here.

Comment (bigfishdesign13) [reply app/javascript/components/Home/Home.jsx]: Why not remove the following `h1` style from the stylesheet?
```
h1 {
    text-transform: capitalize !important;
}
```

**Review by bigfishdesign13 — COMMENTED** [review]
Overall, this looks good.  I just had one quetion.

Also, I see there was one line from the spreadsheet that was not addressed.  Also, I realized that I did not include changes for the `Cancel order` page, so I added a few more lines to the spreadsheet.

Shouldn't there be more than seven pages included in this PR?

File: app/javascript/components/Home/Home.jsx (hunk 1 above)
Comment (gonuguntla) [reply app/javascript/components/Home/Home.jsx:95]: I thought it will effect for all h1 's. I feel class is the best option

**Comment by bigfishdesign13** [comment]
The "Cancel My Order" button label needs to be adjusted.  Otherwise, the updates look good.
image

**Pull request merged**